scripts/validate_target_repo.py

```
@@ -0,0 +1,165 @@
+    # With --target-repo flag
+    python scripts/validate_target_repo.py "feature/path --target-repo ~/my-fork/opendatahub-tests"
+"""
+
+import re
+import sys
+from pathlib import Path
+
+from scripts.parse_skill_args import extract_flag_value
+from scripts.utils.error_utils import exit_error
+
+# GitHub owner: 1–39 chars, alphanumeric or hyphen, cannot start/end with hyphen.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)